docker/luna_tutorial/apply_ports.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+### find open ports and apply to the tutorial build and configuration files
+
+
+# find and apply open port for girder
+echo "Finding available port for girder..."
+PORT=$(./find_open_port.sh 8080 65535)
+echo "Found port $PORT"
+rm -f .dsa_port
+echo $PORT >> .dsa_port
+
+unameOut="$(uname -s)"
+
+if [[ "$unameOut" == *"Darwin"* ]];
```

Comment:
Nice improvement with host/port config. It would be good to add a comment in the nb that host/port is auto generated in the config files.